### DISCUSSION TITLE
Double error message on validator

### DISCUSSION THREAD

**Discussion by vicainelli** [adonisjs/core#2782] [Help] [answered]
Hi all,

I'm facing this error when I was running my tests.

It seems the error messages came in double, but I have no idea why.

```
AdonisJS v5
NodeJS v15.7.0
```

ProductsStoreValidator.ts
```typescript
import { rules, schema } from '@ioc:Adonis/Core/Validator'
import { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'

export default class MenusStoreValidator {
  constructor(protected ctx: HttpContextContract) {}

  public schema = schema.create({
    name: schema.string({ trim: true }, [rules.required(), rules.minLength(3)]),
  })

  public messages = {
    'name.required': 'you must provide a name',
  }
}
```

I'm just print the error object on my test case:
<img width="602" alt="Screenshot 2021-06-13 at 10 55 28" src="https://user-images.githubusercontent.com/1155694/121801101-dee7d380-cc35-11eb-854c-cf7f6f871989.png">

**Comment by thetutlage** [comment, marked as answer]
Fields are required by default and hence there is no need to use `rules.required` method

**Comment by vicainelli** [comment]
Thanks @thetutlage, good to know that :)
